Makefile

```diff
@@ -33,6 +33,8 @@ GOFILES := go list  -f '{{join .Deps "\n"}}' ./cmd/minishift/ | grep $(REPOPATH)
 PACKAGES := go list ./... | grep -v /vendor
 SOURCE_DIRS = cmd pkg test
 
+MINISHIFT_GITHUB_API_TOKEN = $(shell echo $MINISHIFT_GITHUB_API_TOKEN)
```

Comment:
@hferentschik , I am mostly using `GH_TOKEN` variable.

@praveenkumar this has been used here https://github.com/minishift/minishift/blob/master/pkg/util/github/github.go#L80 and need to prevent API rate limit.